src/Modules/taskmanage.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable no-loop-func */
+
+let List = [];
+let isEditing = false;
+let todoEdit = null;
+
+// Save Task List to Local Storage
+
+const storeList = () => {
+  localStorage.setItem('tasklist', JSON.stringify(List));
+};
+
+// Get Task List from Local Storage
+
+const getData = () => {
+  const localFormData = JSON.parse(localStorage.getItem('tasklist'));
+  if (localFormData == null) {
+    List = [];
+  } else {
+    List = localFormData;
+  }
+};
+
+// Edit selected task
+
+const taskEdit = (todo) => {
+  isEditing = true;
+  todoEdit = todo;
+  const desc = document.getElementById('addtodo');
+  desc.value = todo.description;
+  desc.focus();
+};
+
+// Display Task List
+
+const listDisplay = () => {
+  const ListElement = document.getElementById('alltasks');
+  ListElement.innerHTML = '';
+
+  // Delete task (with delete button)
+
+  const cancelTask = (indexID) => {
+    List = List.filter((ind) => ind.index !== indexID);
+    List = List.map((todo, index) => ({
+      completed: todo.completed,
+      description: todo.description,
+      index: index + 1,
+    }));
+    listDisplay();
+  };
+
+  const taskStatus = (todo) => {
+    List = List.map((todoItem) => {
+      if (todoItem.index === todo.index) {
+        return { ...todo, completed: !todo.completed };
+      }
+      return todoItem;
+    });
+    storeList();
+  };
+
+  // Loop for creating Tasks on List
+
+  for (let i = 0; i < List.length; i += 1) {
```

Comment:
[OPTIONAL]
You can use a forEach loop instead of a regular for loop

Reply:
Thank you for the comments, ill certainly look into it!